app/public/css/styles.css

```diff
@@ -696,21 +696,21 @@ p.hey {
   display: none;
 }
 
-.searchable-select-item .multi-asset-item {
+.multi-asset-item {
```

Comment:
I removed `.searchable-select-item` from selectors as I am also reusing component with those classes outside of the `select`